pkgs/applications/networking/browsers/firefox-bin/update.nix

```diff
@@ -60,27 +60,28 @@ writeScript "update-${pname}" ''
 
   # this is a list of sha256 and tarballs for both arches
   # Upstream files contains python repr strings like b'somehash', hence the sed dance
-  shasums=`cat $HOME/shasums | sed -E s/"b'([a-f0-9]{64})'?(.*)"/'\1\2'/ | grep '\.tar\.[a-z0-9]\+'`
+  shasums=`cat $HOME/shasums | sed -E s/"b'([a-f0-9]{64})'?(.*)"/'\1\2'/ | grep '\.\(tar\.[a-z0-9]\+\|dmg\)$' | grep -v mac-EME-free`
 
   cat > $tmpfile <<EOF
   {
     version = "$version";
     sources = [
   EOF
-  for arch in linux-x86_64 linux-i686 linux-aarch64; do
+  for arch in linux-x86_64 linux-i686 linux-aarch64 mac; do
     # retriving a list of all tarballs for each arch
```

Comment:
technically not a tarball for darwin; not sure if worth updating the comment though with these details